app/assets/javascripts/discourse/app/models/group.js

```diff
@@ -34,9 +34,9 @@ const Group = RestModel.extend({
     return automatic ? "automatic" : "custom";
   },
 
-  findMembers(params, refresh) {
```

Comment:
Note this is a breaking change. There's one private plugin I'm aware of that uses this method and will have to be updated. I've sent it to you via PM.

Reply:
I've merged the fix into that plugin (and pinned plugin version), so I'm merging this.